romanisim/apt.py

```diff
@@ -6,8 +6,9 @@
 fraction of what an APT file seems able to do.
 """
 
-import xml
-from xml.etree import ElementTree
+# import xml
+# from xml.etree import ElementTree
```

Comment:
```suggestion
```

No need to leave commented out code floating around. You can find the code in the git history.

Reply:
Cleaned.